### PR TITLE
Update apns_queue.erl

### LINKED ISSUE #91: apns_queue.erl has problem
line 32, line 111, Queue:queue() compile error:
must take out module name!

make[1]: Leaving directory `/home/jiangyang/apns4erl/deps/sync'
 ERLC   apns_queue.erl apns_sup.erl apns_connection.erl apns.erl
src/apns_queue.erl:32: referring to built-in type queue as a remote type; please take out the module name
src/apns_queue.erl:111: referring to built-in type queue as a remote type; please take out the module name

### PR DESCRIPTION
**Pull request by jianjyan**
[Fixes #91] line 32, line 111, has namespace problem

### CODE REVIEW

File: src/apns_queue.erl [hunk 1]
```diff
@@ -108,7 +108,7 @@ code_change(_OldVsn, State, _Extra) ->
 %% Server implementation, a.k.a.: callbacks
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Comment (gadgetci) [inline src/apns_queue.erl]: According to **Elvis**:
> Line 109 is too long: %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%.

Comment (jianjyan) [inline src/apns_queue.erl]: I didn't modified this line, ti's original code

**Comment by elbrujohalcon** [comment]
Oh, now I see @jianjyan you were probably compiling with an older version of Erlang. What version of Erlang/OTP are you using?

**Comment by jianjyan** [comment]
@elbrujohalcon   it's Erlang R16B02 (erts-5.10.3)

**Comment by elbrujohalcon** [comment]
@jianjyan That's the issue then, these types (With module names) were added to comply with the requirements of Erlang/OTP 18.x, where the built-in types are no longer available.
The proper solution to this issue, to allow you to compile the latest release of apns4erl with R16B02 will require much more work than just reverting that change.
I guess we should remove R16* from the list of supported erlang versions in our rebar.config

**Comment by elbrujohalcon** [comment]
_Fixed_ in a different way in #91

**Pull request closed without merging**